package.json

```diff
@@ -15,14 +15,11 @@
     "react": "^18.2.0",
     "react-dom": "^18.2.0",
     "react-helmet": "^6.1.0",
-    "react-scripts": "^5.0.1",
     "web-vitals": "^2.1.4"
   },
   "scripts": {
-    "start": "react-scripts start",
-    "build": "react-scripts build",
-    "test": "react-scripts test",
-    "eject": "react-scripts eject"
+    "start": "vite",
+    "build": "vite build"
   },
```

Comment:
"scripts": {
  "start": "vite",
  "build": "vite build",
  "serve": "vite preview"
},

Reply:
added

Reply:
Hi, this doesn't solve the empty `index.html` page problem after `npm run build`.
This was just a minor change. 

Again, please test the program and make sure npm run build produces correctly before submitting a PR, thanks!